recipes/replit/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "replit" %}
+{% set version = "3.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/replit-{{ version }}.tar.gz
+  sha256: fcea2df361a195ab6a6739ddba429227f36944075f8852e35f77a4031d263ab0
+  patches:
+    - 0001-add-script.patch
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - replit=replit.__main__:cli
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6
+  run:
+    - aiohttp >=3.6.2
+    - flask >=2.0.0
+    - python >=3.6
+    - requests >=2.25.1
+    - typing_extensions >=3.7.4
+    - werkzeug >=2.0.0
+
+test:
+  imports:
+    - replit
+    - replit.audio
+  commands:
+    - pip check
+    - replit --version
```

Comment:
i'd recommend trying more of the expected stuff

Reply:
I'll see what I can add

Reply:
can't figure out what to add here because all 5 commands require connection to a db on repl.it... I pip installed it to test too. All checks passes otherwise @bollwyvl